Integrations/Forescout/Forescout.py

```diff
@@ -0,0 +1,678 @@
+                    err_msg += ''.join([f'\n{code}: {msg}' for code, msg in zip(codes, messages)])
+                return_error(err_msg)
+
+        resp_type = resp_type.casefold()
+        if resp_type == 'json':
+            return res.json()
+        elif resp_type == 'text':
+            return res.text
+        elif resp_type == 'content':
+            return res.content
+        else:
+            return res
+
+    except requests.exceptions.ConnectionError:
+        err_msg = 'Connection Error - Check that the Server URL parameter is correct.'
+        return_error(err_msg)
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def test_module():
+    """
+    Performs basic get request that requires proper authentication
+    """
+    login()
+    demisto.results('ok')
+
+
+def get_host(args):
+    identifier = args.get('identifier', '')
+    fields = args.get('fields', '')
+    url_suffix = '/api/hosts/'
+    id_type, *ident = identifier.split('=')
+    id_type = id_type.casefold()
+    if len(ident) != 1 or id_type not in {'id', 'ip', 'mac'}:
+        err_msg = 'The entered endpoint identifier should be prefaced by the identifier type,' \
+            ' (\'ip\', \'mac\', or \'id\') followed by \'=\' and the actual ' \
+            'identifier, e.g. \'ip=123.123.123.123\'.'  # disable-secrets-detection
+        raise ValueError(err_msg)
+
+    if id_type == 'ip':
+        # API endpoint format - https://{EM.IP}/api/hosts/ip/{ipv4}?fields={prop},..,{prop_n}
+        url_suffix += 'ip/'
+    elif id_type == 'mac':
+        # API endpoint format - https://{EM.IP}/api/hosts/mac/{mac}?fields={prop},..,{prop_n}
+        url_suffix += 'mac/'
+    # if id_type == 'id' don't change url_suffix -it's already in desired format as shown below
+    # API endpoint format - https://{EM.IP}/api/hosts/{obj_ID}?fields={prop},..,{prop_n}
+
+    url_suffix += '='.join(ident)
+    params = {'fields': fields} if fields != '' else None
+    headers = create_web_api_headers()
+    response = http_request('GET', url_suffix, headers=headers, params=params, resp_type='response')
+    return response
+
+
+def get_host_command():
+    args = demisto.args()
+    identifier = args.get('identifier', '')
+    requested_fields = argToList(args.get('fields', ''))
+    response = get_host(args)
+    data = response.json()
+    host = data.get('host', {})
+    fields = host.get('fields', {})
+
+    if requested_fields and set(requested_fields).issubset(HOSTFIELDS_TO_INCLUDE.keys()) or not requested_fields:
+        included_fields = {
+            HOSTFIELDS_TO_INCLUDE.get(key): val for key, val in fields.items() if key in HOSTFIELDS_TO_INCLUDE.keys()
+        }
+        included_fields_readable = {
+            HOSTFIELDS_TO_INCLUDE.get(key): dict_to_formatted_string(val)
+            for key, val in fields.items() if key in HOSTFIELDS_TO_INCLUDE.keys()
+        }
+    elif requested_fields:
```

Comment:
you already check `requested_fields` in the condition

Reply:
yes but in the previous condition, the expression only evaluates to `True` if there are `requested_fields` _and_ if `requested_fields` are a subset of `HOSTFIELDS_TO_INCLUDE` keys.